x/concentrated-liquidity/types/constants.go

```diff
@@ -12,5 +17,11 @@ var (
 	MaxSqrtRatio = sdk.MustNewDecFromStr("18446050711097703529.7763428")
 	// TODO: this is a temp value, figure out math for this.
 	// we basically want getSqrtRatioAtTick(MIN_TICK)
-	MinSqrtRatio = sdk.MustNewDecFromStr("0")
+	MinSqrtRatio = GetMinSqrtRatio()
```

Comment:
We should hardcode this value instead

Reply:
Yeah, we can just have GetMinSqrtRatio take an input, in this case of -887272